### PR TITLE
Solve problem APC ext php >= 7

### PR DESCRIPTION
**Pull request by gplaza**
pecl install apc seem deprecated, apcu alone is not compatible with apc used in yii2, solution : use apcu with backward compatibility

### CODE REVIEW

File: docs/install-extensions.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ RUN apt-get update && \
     RUN pecl install apc
     RUN echo "extension=apcu.so" > /usr/local/etc/php/conf.d/pecl-apcu.ini
 
+### APC (>= PHP 7)
+
+    RUN pecl install apcu_bc
+    RUN echo "extension=apcu.so" > /usr/local/etc/php/conf.d/docker-php-ext-apc1.ini
+    ,RUN echo "extension=apc.so" > /usr/local/etc/php/conf.d/docker-php-ext-apc2.inii
```
Comment (schmunk42) [inline docs/install-extensions.md:33]: the comma and i at the end look like a typo to me

**Review by schmunk42 — CHANGES_REQUESTED** [review]

File: docs/install-extensions.md (hunk 1 above)
Comment (gplaza) [reply docs/install-extensions.md:33]: yes, sorry there is a typo ...

**Comment by schmunk42** [comment]
There's still an additional `i` at the end ... `inii`.

And did you test this? So it works by loading both extensions?
I did not find installation instructions here: https://github.com/krakjoe/apcu-bc

**Comment by gplaza** [comment]
> There's still an additional `i` at the end ... `inii`.
> 
> And did you test this? So it works by loading both extensions? I did not find installation instructions here: https://github.com/krakjoe/apcu-bc

yes, i have test. i work by loading both expensions, but : i need to load in this exactly order, fisrt one : apcu.so, second : apc.so
in the other order :  error "zif_apcu_store" see : https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=909752

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**